data/remnant/remnant ships.txt

```diff
@@ -230,6 +230,324 @@ ship "Albatross" "Albatross (Ember Shield)"
 	turret
 
 
+
+ship "Modified Dromedary"
+	sprite "ship/modified dromedary"
+	thumbnail "thumbnail/modified dromedary"
+	attributes
+		licenses
+			Remnant Capital
+		category "Transport"
+		"cost" 57600000
+		"shields" 15100
+		"hull" 9600
+		"required crew" 95
+		"bunks" 300
+		"mass" 2140
```

Comment:
I don't know the mathematical basis for this mass choice, but the mass I set for it *feels* really good to me. Halving the weight starts to feel really light and insubstantial for this ship.

Reply:
Maybe its just a matter of giving it worse engines then?

Reply:
No, less mass and less engines don't give the same feel (and don't give the same power draw or respond to afterburners and other applications of force) as having high mass does. 

Putting it another way: There's a reason why the Bactrian is the sole city-ship remaining in production. While I can't speak to the others, the Dromedary ceased to be produced because it was so massive that it was much harder to get it to manoeuvre like the Bactrian can. The Dromedary has - just looking at the hull mass itself - almost double the mass per unit of engine space as the Bactrian. This is intentional. Whereas the Bactrian is, in city terms, an army base town; the Dromedary is an industrial city.

Especially in its modified form as seen here. When it says "modified", it's probably better to read "augmented" or "added-on-to", as while its captain did replace some stuff, they moved on very quickly to just tacking on *more* stuff. 

The original Dromedary would definitely have lower mass. Maybe something in the 1600 range. But this modified one is not that.

Reply:
We're looking to fiddle with ship masses soon anyway, so I'll let this one stick around for now.

Reply:
Where would the SL have gotten 540 tons of material from in the Secondary Weft? Were they mining asteroids in there, or just making do with what they brought?